MAINTAINERS.txt

```diff
@@ -0,0 +1,30 @@
+The project is currently managed by Justin Cappos at New York University.
+Please see GOVERNANCE.md for the project's governance and maintainership.
+
+Consensus Builder:
+
+  Justin Cappos
```

Comment:
I'm not sure how much the NJIT folks contribute to the specification, but should they also be listed here? 
 
The NJIT is acknowledged in one of the other docs of this pull request.

Reply:
We contributed to the spec. 
NJIT is mentioned in the ACKNOWLEDGMENTS.md file, but am not sure what's the difference between that file and this file.

Reply:
The acknowledgments contains the involved institutions, whereas the maintainers file lists the individual persons (plus roles and contact details) that participate in the project.

Since the governance file is meant to cover the reference implementation and the specification, we (or actually the consensus builder) can make any contributor, who expresses interest, a maintainer.

Reply:
@reza-curtmola, I will merge this PR for now, but we can revise the document at any time.